CMakeLists.txt

```diff
@@ -139,14 +168,20 @@ if (MATERIALX_BUILD_GEN_ARNOLD)
 endif(MATERIALX_BUILD_GEN_ARNOLD)
 
 if (MATERIALX_BUILD_RENDER)
+    add_definitions(-DMATERIALX_BUILD_RENDER)
+    add_subdirectory(source/MaterialXRender)
+    if (MATERIALX_BUILD_RENDEROSL)
+        add_subdirectory(source/MaterialXRenderOsl)
+        add_definitions(-DMATERIALX_BUILD_RENDER_OSL)
```

Comment:
Possibly doesn't matter, but should the definitions be added before the subdirectories in case they need the definitions?

Reply:
It doesn't matter it adds it to the current directory (root) and below.